server/src/db/queries/user/getStepStatus.js

```diff
@@ -0,0 +1,14 @@
+const Query = require('../../core/Query')
+
+
+const getUserStepStatus = ({username, stepId}) => {
+    return new Query({
+        statement: `
+        MATCH (u:User {username: $username})-[r]->(s:Step {id: $stepId})
+        RETURN type(r)
+        `,
+        params: {username, stepId}
```

Comment:
Follow style, don't use the shorthand syntax. If you're gonna use it, then make sure it is also used everywhere else.